pkg/metrics/unique_visits.go

```diff
@@ -0,0 +1,49 @@
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"go.uber.org/zap"
+)
+
+type uniqueVisits struct {
+	counter       prometheus.Gauge
+	visitorsCache map[string]struct{} // TODO: We may want to make this persistent in the DB.
+}
+
+// Visits
+var visitCountPerWeek = "visits_count_per_week"
+
+const maxCacheSize = 10000
```

Comment:
not really helpful. At present, it's not useful because it's pretty obvious that we don't hit this limit, and in the long term, we would like to have the real number of unique visits. 
If we really pass 10k I think we should find better solution.